components/text-utilities/text-utilities.css

```diff
@@ -0,0 +1,97 @@
+
+/* ----------------- headings -------------------- */
+
+.h1, 
+.h2, 
+.h3, 
+.h4, 
+.h5, 
+.h6 {
+    margin: 0.6rem 0;
+    line-height: 1.2;
+    font-weight: 500;
+}
+
+
+.h1,
+.text-xxl {
```

Comment:
why use both class name && element name for selection?

Reply:
I was adding style to selector as well as class, but that is not  a good strategy